.travis.yml

```diff
@@ -2,14 +2,15 @@ language: groovy
 
 jdk:
     - oraclejdk8
-    
-before_install:
-    - sudo apt-get update -qq
-    - sudo apt-get install oracle-java8-installer
+
+# Use container build
+sudo: false
+
+before_script:
     - export GRADLE_OPTS='-Dorg.gradle.parallel=false'
```

Comment:
@Zomis Why is the build not run in parallel? Added https://github.com/Cardshifter/Cardshifter/commit/8777b5fb1c33afc08c43a8b1da555c2cc82cb4c1

We could probably speed it up by not setting this option:

> The new containers have 2 dedicated cores and 4GB of memory, vs 1.5 cores and 3GB on our legacy infrastructure. CPU resources are now guaranteed, which means less impact from ‘noisy neighbors’ on the same host machine and more consistent build times throughout the day.

(http://docs.travis-ci.com/user/migrating-from-legacy/)